### PR TITLE
Clarify purpose of FMS4C contact form

### LINKED ISSUE #1553: Add additional notice to /councils contact page
We’ve had a few of these requests in the past few days. I think we need to add something to say ‘if you wish to report a problem please click here’ to take them to the main screen or possibly to ask for their location and jump into the process from there.

### PR DESCRIPTION
**Pull request by zarino**
* Tweak wording to make it clearer that the form is unsuitable for residents wanting to report street problems.
* Record HTTP Referer when form is submitted, so we can see which pages are leading people to the FMS4C page.
* Tidy up capitalisation elsewhere in headings on the page.
* Replace links to `/council` with `/about/council` to avoid an unnecessary redirect (thanks @davea!) and remove the `www.fixmystreet.com` prefix inside the fixmystreet.com cobrand templates.

Fixes #1553.

screen shot 2017-02-16 at 15 59 04

### CODE REVIEW

File: templates/web/fixmystreet.com/footer_extra.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h2 class="mysoc-footer__site-name">FixMyStreet</h2>
                             %]>[% loc("Local alerts") %]</[% c.req.uri.path == '/alert' ? 'span' : 'a' %]></li>
                     </ul>
                     <ul>
-                        <li><a href="https://www.fixmystreet.com/council">FixMyStreet for Councils</a></li>
+                        <li><a href="/council">FixMyStreet for Councils</a></li>
```
Comment (davea) [inline templates/web/fixmystreet.com/footer_extra.html]: Could this (and the others) be to `/about/council`, as that's where the `/council` redirect takes you?

File: templates/web/fixmystreet.com/about/council.html [hunk 2]
```diff
@@ -245,20 +258,8 @@ <h2 class="councils-hero__title">Reliable fault-reporting and case management fo
...
-          <label><input name="extra.product" id="product" type="radio" value="case-management-integration">Case-management integration</label>
-          <label><input name="extra.product" id="product" type="radio" value="top-to-bottom-case-management">Top-to-bottom case-management</label>
-        </div>
-      </div>
-      <div class="form-group">
-        <label for="message">Message</label>
-        <textarea name="message" id="message" required> </textarea>
-      </div>
       <div class="form-group submit-group">
+        <input type="hidden" name="extra.referer" value="[% c.req.headers.referer %]">
```
Comment (davea) [inline templates/web/fixmystreet.com/about/council.html]: This needs a `| html` so the value is escaped.

File: templates/web/fixmystreet.com/footer_extra.html (hunk 1 above)
Comment (zarino) [inline templates/web/fixmystreet.com/footer_extra.html]: I wondered about that! If we change these to `/about/council`, should we also change all the other instances of `www.fixmystreet.org/council` in all the other cobrands?

Comment (davea) [inline templates/web/fixmystreet.com/footer_extra.html]: Saving the web, one redirect at a time :) Yep, it wouldn't hurt 👍

Comment (zarino) [inline templates/web/fixmystreet.com/footer_extra.html]: Ok! Will pull out the two fixed URLs from the existing commit into a new separate commit, and then add in fixes for all the other URLs.

**Pull request merged**